client-java/controller/src/main/java/org/evomaster/client/java/controller/problem/rpc/CodeJavaGenerator.java

```diff
@@ -0,0 +1,340 @@
+package org.evomaster.client.java.controller.problem.rpc;
+
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * a set of util for generating instance creation and assertion with java in tests
+ * in order to make endpoint invocation
+ */
+public class CodeJavaGenerator {
```

Comment:
why was this necessary? i mean, why need of Java code on the driver? 
can't the API be called via reflection?

Reply:
this class is not related to endpoint invocation
it is only used to generate test scripts and assertion scripts in generated tests.